app/webroot/css/layouts/elements.css

```diff
@@ -736,42 +736,56 @@
 /*
  * Pagination
  */
+
 .paging {
     margin: 20px 0;
     text-align: center;
 }
 
-.paging a {
-    padding: 3px 5px;
-    margin: 2px;
-    background: #D3F3B9;
-    color: #257D0C;
+.numbers a {
+    padding: 5px 8px;
+    margin: 3px;
+    background: #EEEEEE;
+    color: #757575;
 }
 
-.paging a:hover {
-    background: #257D0C;
-    color: white;
+.numbers a:hover {
+    background: #E0E0E0;
+    color: #757575;
 }
 
 .paging .disabled {
     display: inline;
     padding: 3px 5px;
     margin: 3px;
```

Comment:
The `display`, `padding` and `margin` can be deleted.
* `display: inline` doesn't have any effect
* the padding and the margin are causing misalignment with the disabled arrow icons